ldk/node/examples/selfTestLoop/src/testingFixtures/testSuite.ts

```diff
@@ -12,12 +17,56 @@ export default class TestSuite {
     this.logger = logger;
   }
 
-  public async start(host: HostServices): Promise<void> {
+  public async start(host: HostServices): Promise<boolean> {
+    const elements = {} as Element;
+    let i = 0;
     for await (const test of this.tests) {
-      test.runTest(host, this.logger);
+      try {
+        this.logger.info('Before next test');
+        await test.runTest(host, this.logger);
+        elements[`${i}`] = {
+          value: test.getStatus(),
+          label: test.getId(),
+          order: i + 1,
+          type: 'pair',
+        };
+        i += 1;
+      } catch (err) {
+        elements[`${i}`] = {
+          value: test.getStatus(),
+          label: test.getId(),
+          order: i + 1,
+          type: 'pair',
+        };
+        i += 1;
+      }
     }
-    return new Promise((resolve) => {
-      resolve();
+
+    const hotkeys = {
+      key: '/',
+      modifiers: {
+        ctrl: true,
+      },
+    };
+
+    const listWhisper = host.whisper.listWhisper({
+      label: 'Self Test - Results',
+      markdown: '',
+      elements,
+    });
+    const markdown = host.whisper.markdownWhisper({
+      label: 'Self Test - Results',
+      markdown: 'Press "Ctrl + /" to bring back up the original whisper',
+    });
+    const keyboard = host.keyboard.streamHotKey(hotkeys, (error, response) => {
+      if (error) {
+        // Do nothing
```

Comment:
Is there a reason we just toss the error?

Reply:
This specific listener is to get back to the starting menu to pick which group you want to test. My thinking here being that if the tests works, this should work too. So why expect an error to be thrown?